app/components/find/courses/entry_requirements_component/view.html.erb

```diff
@@ -52,19 +52,23 @@
         <%= helpers.markdown(course.additional_gcse_equivalencies) %>
       </p>
     <% end %>
-  </div>
 
-  <% if course.personal_qualities.present? %>
-    <h3 class="govuk-heading-m">Personal qualities</h3>
-    <div data-qa="course__personal_qualities">
-      <%= helpers.markdown(course.personal_qualities) %>
-    </div>
-  <% end %>
+    <h3 class="govuk-heading-m">Qualifications gained outside the UK</h3>
```

Comment:
I would prefer that we start using locales consistently and move this text to a locale file. Given that we are making lots of changes to these pages in this epic, I think now is the time to start doing that -- starting as we mean to go on. What do you think?

Reply:
Yep, sounds good